Sources/Purchasing/StoreKitAbstractions/SK1StoreProductDiscount.swift

```diff
@@ -75,7 +75,7 @@ private extension StoreProductDiscount.PaymentMode {
         case .freeTrial:
             self = .freeTrial
         @unknown default:
-            Logger.appleWarning(Strings.storeKit.skunknown_payment_mode(String.init(describing: paymentMode)))
+            Logger.appleWarning(Strings.storeKit.skunknown_payment_mode(.init(describing: paymentMode)))
```

Comment:
Can we do
```suggestion
            Logger.appleWarning(Strings.storeKit.skunknown_payment_mode(paymentMode.rawValue))
```

That doesn't use reflection so it's a lot faster

Reply:
Sadle it's a `UInt` enum :(

Reply:
Then `String(enum.rawValue)`. Describing uses runtime reflection and it's A LOT slower.

Reply:
You already merged so it's fine. It was already like this anyway.

Reply:
And it's not like this is a hot path.